storage/google/cloud/storage/_helpers.py

```diff
@@ -94,6 +99,8 @@ def reload(self, client=None):
         # Pass only '?projection=noAcl' here because 'acl' and related
         # are handled via custom endpoints.
         query_params = {'projection': 'noAcl'}
+        if self.user_project is not None:
```

Comment:
We're not worried about a `NotImplementedError` here (i.e. all implementers of this base class will implement this property)?